packages/date/src/DatepickerInput.spec.tsx

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-global-assign -- need for test purposes */
+import React from 'react';
+
+import { render, configure } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import timezonedDate from 'timezoned-date';
+
+import { DatepickerInput } from './DatepickerInput';
+import { userInputFromDatetime } from './utils/date';
+
+configure({
+  testIdAttribute: 'data-test-id',
+});
+
+let originalDate: typeof Date;
+beforeEach(() => {
+  originalDate = Date;
+});
+afterEach(() => {
+  Date = originalDate;
+});
+
+const renderDatepicker = (dateString: string) => {
+  const { date } = userInputFromDatetime({
+    value: dateString,
+    uses12hClock: false,
+  });
+
+  return render(<DatepickerInput value={date} onChange={jest.fn()} />);
+};
+
+describe('Date: DatepickerInput', function () {
+  describe('renders correct date, when the system UTC offset is +02:00 and the one in the props.value is', () => {
+    it('+04:00', async () => {
+      Date = timezonedDate.makeConstructor(120);
```

Comment:
Does this work without setting `Date` in the global scope? The way I read it here we have to do that to hook into the date object properly? https://github.com/thorn0/timezoned-date

Reply:
it works you can check it by logging the date. But I can change it to `global.Date` if it's gonna be more clear

Reply:
Nah no worries, thanks for checking 💯